core/backend/src/IModelDb.ts

```diff
@@ -1366,8 +1366,9 @@ export namespace IModelDb { // eslint-disable-line no-redeclare
       return this.tryGetModelJson({ id });
     }
 
-    /** Query for the last modified time of the specified Model.
-     * @internal
```

Comment:
Before we make methods like this @public, should we make them take named arguments?

missing “if” in comment.

Reply:
Added "if".
I don't really want to encourage proliferation/expansion of these convenience APIs. If people want to execute a more complicated query for last modified time than "by id", they can write ECSql to do it.

Reply:
Yeah, that’s probably why it was @internal to begin with. But you only get one chance to set something in stone..